MediaBrowser.Controller/Entities/UserItemData.cs

```diff
@@ -12,6 +12,13 @@ namespace MediaBrowser.Controller.Entities
     /// </summary>
     public class UserItemData
     {
+        /// <summary>
+        /// The _rating.
+        /// </summary>
+        private double? _rating;
+
+        public const double MinLikeValue = 6.5;
```

Comment:
Const values should come before fields

Reply:
Fixed in https://github.com/jellyfin/jellyfin/pull/6423/commits/e7a3552aae556a5afa58207db353f10de9021274.